.github/workflows/preDeploy.yml

```diff
@@ -105,6 +101,13 @@ jobs:
           WORKFLOW: cherryPick.yml
           INPUTS: '{ "PULL_REQUEST_NUMBER": "${{ needs.chooseDeployActions.outputs.mergedPullRequest }}", "NEW_VERSION": "${{ env.NEW_VERSION }}" }'
 
+      # Note: we need to create this tag but not push it, because GitUtils.getPullRequestsMergedBetween uses tags to generate a list of merged pull requests
```

Comment:
Thanks for working on this comment! I think there is still some room for improvement though. It needs to explain why the tag isn't being pushed, and I think it should also explain how tags are used in getPullRequestsMergedBetween. Something like this:

> Create a tag for this version so that GitUtils.getPullRequestsMergedBetween can generate a list of merged pull requests that happened between two release version tags. It's not necessary to push this tag because... XYZ.

Reply:
That's great feedback! One of my goals for the GHA code is to make it more approachable to new contributors, and there are definitely a few "gotchas" hiding in here. So it's definitely good to explain this one in more detail :)

I updated the comment, let me know what you think!

Reply:
I see 2 other places where the old comment still exists, can you update those as well?

Reply:
Updated again!